man/sar.in

```diff
@@ -348,16 +348,21 @@ Set the ending time of the report. The default ending time is
 18:00:00. Hours must be given in 24-hour format.
 This option can be used when data are read from
 or written to a file (options -f or -o).
-.IP -F
+.IP "-F [ MOUNT ]"
```

Comment:
Parameter MOUNT should also appear in the SYNOPSIS section at the beginning of the man page.